src/99-appendices/08-coordinate-systems.md

```diff
@@ -112,21 +112,28 @@ below, the origin is at the AC and the orientation of the axes is RAS. Unless
+| NIHPD                                               | Pediatric templates generated from the NIHPD sample. Available for different age groups (4.5–18.5 y.o., 4.5–8.5 y.o., 7–11 y.o., 7.5–13.5 y.o., 10–14 y.o., 13–18.5 y.o. This template also comes in either -symmetric or -asymmetric flavor. [http://www.bic.mni.mcgill.ca/ServicesAtlases/NIHPD-obj1](http://www.bic.mni.mcgill.ca/ServicesAtlases/NIHPD-obj1)                                                                                                                                                       |
+| OASIS30AntsOASISAnts                                | [https://figshare.com/articles/ANTs_ANTsR_Brain_Templates/915436](https://figshare.com/articles/ANTs_ANTsR_Brain_Templates/915436)                                                                                                                                                                                                                                                                                                                                                                                     |
+| OASIS30Atropos                                      | [https://mindboggle.info/data.html](https://mindboggle.info/data.html)                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| Talairach                                           | Piecewise linear scaling of the brain is implemented as described in TT88. [http://www.talairach.org/](http://www.talairach.org/)                                                                                                                                                                                                                                                                                                                                                                                      |
+| UNCInfant                                           | Infant Brain Atlases from Neonates to 1- and 2-year-olds. [https://www.nitrc.org/projects/pediatricatlas](https://www.nitrc.org/projects/pediatricatlas)                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
how will the temporal qualifiers be noted for this atlas and for NIHPD?

Reply:
In TemplateFlow we are using a `cohort-` entity for that. However, I don't think age ranges should be considered as generators of new spaces.

Reply:
this may boil down to the question whether a custom template is a new space? 

i think here `space` is more related to image landmarks being different rather than whether coordinate axes and units are aligned.

so from that perspective age-ranges could create different landmarks.

Reply:
I wouldn't go down that road. Coordinates across all age ranges are the same (i.e., the origin of coordinates is always the same and one mm is always the same length too).

For the purpose of registration, you will always have the `cohort-` present in the `ReferenceVolume` (or whatever keyword we end up choosing in the derivatives PR), so the reference will be univocal and you'll be always aware of what range was used.

Reply:
i'm generally ok, but the image coordinates are different from the coordinate system. under coordinate system, location of origin, units, and angle of axes all the MNI spaces are essentially the same.

i think we just need to be careful in what we call landmarks and coordinate systems. i think robert and i had an extensive discussion on this that led to the updated language for coordinates/landmarks/etc.,..

Reply:
> i'm generally ok, but the image coordinates are different from the coordinate system.

Not sure how this applies to this PR, could you clarify?

> under coordinate system, location of origin, units, and angle of axes all the MNI spaces are essentially the same.

Agreed.

> i think we just need to be careful in what we call landmarks and coordinate systems.

Agree - but this statement makes me believe you are not completely sure about it in the context of this PR, is that correct? could you clarify where you're getting at?

> think robert and i had an extensive discussion on this that led to the updated language for coordinates/landmarks/etc.,..

hehehe I witnessed part of that :D